### ISSUE TITLE
Unable to launch app on iOS simulator using Appium 1.9.1

### ISSUE THREAD

**Issue by kulswapnil** [appium/appium#11920]
## The problem

I am trying to run tests on an iOS simulator(12.1) with Appium desktop 1.9.1 .
Appium installs the app and WDA, but the app is not launching.

## Environment

* Appium version (or git revision) that exhibits the issue: 1.9.1
* Last Appium version that did not exhibit the issue (if applicable): NA
* Desktop OS/version used to run Appium: 10.14.2
* Node.js version (unless using Appium.app|exe): v11.6.0
* NPM version: 6.5.0
* Mobile platform/version under test: 12.1
* Real device or emulator/simulator: Simulator
* Appium CLI or Appium.app|exe: Appium.app

## Details

With this primary set up, Appium installed the app & WDA on the simulator, but couldnt launch the app. App opened up & closed immediately. 

## Link to Appium logs

Appium logs - https://gist.github.com/kulswapnil/6a0ed73a0aa5f90f84e671446d116917

**Comment by mykola-mokhnach** [comment]
> [warn] [35m[XCUITest][39m Simulator architecture appears to be unsupported by the '/var/folders/wq/zbtfrs815mv9dsvy2sv722p1kzxg9v/T/20181129-1495-ehqhri.5u1df/Payload/DASH-Dev.app' application. Make sure the correct deployment target has been selected for its compilation in Xcode.
> [warn] [35m[XCUITest][39m Don't be surprised if the application fails to launch.

Closed as duplicate

**Comment by kulswapnil** [comment]
can i get the link for original issue?

**Comment by mykola-mokhnach** [comment]
Updating appium won't help. You must get the correct build from developers

**Comment by kulswapnil** [comment]
Hu mykola, can you please provide me the link/issue number for the original issue? Also i tried to install a sample app, that also failed to launch.

**Comment by kulswapnil** [comment]
I have installed WDA manually & tried to launch the app. I got following error.
t =  1949.28s Open com.ibm.Altoro-App
    t =  1949.28s     Launch com.ibm.Altoro-App
2019-01-01 17:57:33.823284-0500 WebDriverAgentRunner-Runner[11562:122072] Enqueue Failure: Failed to launch com.ibm.Altoro-App: The operation couldn’t be completed. (FBSOpenApplicationErrorDomain error 1.) /Applications/Appium.app/Contents/Resources/app/node_modules/appium-xcuitest-driver/WebDriverAgent/WebDriverAgentRunner/UITestingUITests.m 37 1
